### PR TITLE
Custom modules

### PR DESCRIPTION
**Pull request by maxim8898**

### CODE REVIEW

File: web/modules/custom/se_sso_idms/src/IdmsRequest.php [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+  protected $max_redirects;
+  protected $timeout;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function __construct(array $vars) {
+    //$this->root_url = _se_template_integration_exch_url(['path']);
+    // TODO Add variable to state.
+    $this->root_url = 'https://exst-ui-uat.schneider-electric.com';
```
Comment (gunosov) [inline web/modules/custom/se_sso_idms/src/IdmsRequest.php]: I propose not use hardcoded string values.

Comment (maxim8898) [reply web/modules/custom/se_sso_idms/src/IdmsRequest.php]: Thanks. It is temporary hardcode for local env.

File: web/modules/custom/se_template_integration/theme/css/index.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            <div class="row">
+                <div class="column" style="background-color:#e7e7e7;">
+                    <a
+                       style="text-decoration:none"
+                       href="/contact">
+                        <div class="icon"><img
+                            src="/files/c1.png?v=1558470118000"
+                            width="30"
+                            height="22"><p class="link2"
+                           style=" padding-top: 10px">Contact US</p>
```
Comment (gunosov) [inline web/modules/custom/se_template_integration/theme/css/index.html]: Inline styles not good solution

File: web/modules/custom/se_template_integration/theme/templates/se-footer.html.twig [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<div class="se-customization-wrapper footer-wrapper">
+    <div class="container-width">
+        <div class="footer-container">
+            <nav class="footer-nav">
+                    <a class="link" href="{{ root_url }}/about">About Us</a>
```
Comment (gunosov) [inline web/modules/custom/se_template_integration/theme/templates/se-footer.html.twig]: Think about internationalization

File: web/modules/custom/se_template_integration/theme/templates/se-header-block-not-auth.html.twig [hunk 4]
```diff
@@ -0,0 +1,37 @@
+<div class="se-customization-wrapper header-wrapper">
+    <div class="preheader-section">
+        <div class="preheader-container container-width">
+            <a class="logo-link" href="https://www.schneider-electric.com">
+                <img alt="Schneider Electric" class="logo-size" src="https://exst-ui-uat.schneider-electric.com/assets/img/green-logo.svg">
```
Comment (gunosov) [inline web/modules/custom/se_template_integration/theme/templates/se-header-block-not-auth.html.twig]: Use variable instead hardcode

File: web/modules/custom/se_template_integration/theme/templates/se-secondary-menu.html.twig [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+ *     menu tree.
+ *   - is_collapsed: TRUE if the link has children within the current menu tree
+ *     that are not currently visible.
+ *   - in_active_trail: TRUE if the link is in the active trail.
+ *
+ * @ingroup themeable
+ */
+#}
+{% import _self as menus %}
+<div>555</div>
```
Comment (gunosov) [inline web/modules/custom/se_template_integration/theme/templates/se-secondary-menu.html.twig]: 555?

**Pull request closed without merging**